share/instruments/instruments.xml

```diff
@@ -126,8 +126,25 @@
                   <transposeDiatonic>8</transposeDiatonic>
                   <transposeChromatic>13</transposeChromatic>
                   <Channel>
-                        <program value="72"/>
+                        <program value="72"/> <!--Piccolo-->
+                  </Channel>
+            </Instrument>
+            <Instrument id="eb-piccolo">
+                  <longName>E♭ Piccolo</longName>
+                  <shortName>E♭ Picc.</shortName>
+                  <description>Piccolo Flute in E♭</description>
+                  <musicXMLid>wind.flutes.flute.piccolo</musicXMLid>
+                  <transposingClef>G</transposingClef>
+                  <concertClef>G8va</concertClef>
+                  <barlineSpan>1</barlineSpan>
+                  <aPitchRange>77-108</aPitchRange>
+                  <pPitchRange>77-111</pPitchRange>
+                  <transposeDiatonic>9</transposeDiatonic>
+                  <transposeChromatic>15</transposeChromatic>
+                  <Channel>
+                        <program value="72"/> <!--Piccolo-->
                   </Channel>
+                  <genre>concertband</genre>
```

Comment:
E-flat piccolo is even rarer than D-flat piccolo, which is already extremely rare nowadays. So I don't recommend putting it in any genre too.

Reply:
thie is the request though, "_I have come across some concert band pieces which, among other things, call for the E-flat piccolo_"

Reply:
But again, I have seen C Clarinet in _some_ orchestral pieces too (Beethoven 5th Symphony Mvt. 4), but we don't list it in the orchestra genre :-)

Reply:
I agree with Howard and not putting all exotic instruments in any template. It is also in line with we do now. E.g. there is no bass saxophone in the concert band template while it is a much more common instrument as Eb piccolo.
I think it is better to limit the template to more common instruments (although for e.g. a concert this vay per county!). Exotic instruments can be added later or users can make their own template.

Reply:
It is not in any template. 
It is in the concert band genre, because that was the request being made. Discuss it there, not here please.

Reply:
It is not in any template and this PR is not about templates at all.
And it is not in the 'common' genre either, so doesn't clutter the list.
It is in the concert band genre, because that was the request being made. Discuss it there, not here please.
If C Clarinet is used in orchestral pieces, it should be listed in that genre.

Reply:
Yes it is exotic. Orchestras are becoming more and more common nowadays. This is not classicism or romanticism era anymore, I would say using clarinets other than B-flat or A is a mistake, unless the part is very difficult to be played by other clarinets (which is almost impossible) or the composer wishes the piece to have the special sound quality of a certain clarinet (which is not common either). I would even go so far to say that a professional composer, unless being very fastidious, can only have the opportunity to use C Clarinet no more than five times. I don't know about concert band, but I cannot imagine Piccolo in E-flat being used more than C Clarinet. If that's so, why bother having instruments like this being listed in any genre?

Reply:
If you're don't know about concert bands, why opposing this instrument getting added into that genre, when exactly that was the request? Better discuss that with the requester, in the forum topic

Reply:
> It is in the "concertband" genre, because that was the request being made.

Plus, I don't see a request like that. The OP only said he'd came across piccolo in E-flat in some concert band pieces, that doesn't mean we should put it in the concert band genre.

Reply:
So to me exactly there is the request,  _came across piccolo in E-flat in some concert band pieces_ So to my understanding this is, at least occasionally, a concert band instrument. Again, discuss this there, not here, please.

Reply:
> If you're don't know about concert bands

I mean, I don't know about concert band scores, but I've never heard of the flute/piccolo players around me mentioning E-flat piccolo. There have been some times when they mentioned D-flat piccolo, but E-flat piccolo has never been mentioned. Also search the internet and see how many times it is mentioned. Also think about why for more than ten years this instrument has only been mentioned recently. I'm not opposing the instrument, I'm just persistent that genres aren't the place to be cluttered.

Reply:
> Again, discuss this there, not here, please.

The reason why I'm discussing it here is because on contrary, I don't think the thing we're talking about is relevant to the original post. To me, "came across piccolo in E-flat in some concert band pieces" is not the same thing as and doesn't indicate "please add piccolo in E-flat to classic band genre".

Reply:
And just like a mistake which happened some time ago (and got fixed by me), instead of the regular Trumpet in B-flat, the Piccolo Trumpet in B-flat was added to orchestra genre. What good do you see in this? If a composer doesn't know or doesn't notice the difference, he may use it whereas what they want is a regular non-piccolo trumpet, thus causing confusion. If a composer wants to start a piece in E-flat major, they see a piccolo in E-flat in concert band genre and they say "cheers, let's use that", while a lot of (if not "most") piccolo players don't even have a piccolo in E-flat.

Reply:
Again, and for the last time, discuss that with the requester in the forum topic.

Reply:
Wrong accidental usage doesn't count as an argument for hide an instrument. But a concert band knowledgeable person's opinion (like that fellow of your's) does :-)
Sorry for having been difficult here, but I don#t know anything about concert bands and you admitted to not knowing much about it either, so the original request was all I had to go on.